src/mds/MDSRank.cc

```diff
@@ -2385,11 +2385,9 @@ void MDSRankDispatcher::handle_mds_map(
     dout(0) << "WARNING: inline_data support has been deprecated and will be removed in a future release" << dendl;
 
   if (scrubstack->is_scrubbing()) {
-    unsigned active = mdsmap->get_num_mds(MDSMap::DaemonState::STATE_ACTIVE);
-    unsigned stopping = mdsmap->get_num_mds(MDSMap::DaemonState::STATE_STOPPING);
-    if (active + stopping > 1) {
-      C_MDSInternalNoop cond;
-      scrubstack->scrub_abort(&cond);
+    if (mdsmap->get_max_mds() > 1) {
+      auto c = new C_MDSInternalNoop;
```

Comment:
@batrick Won't this be a memory leak ?

Reply:
No, when the context is finished it destroys itself. See `src/include/context.h`.